docs/agent/cli.mdx

```diff
@@ -508,7 +508,7 @@ ngrok tcp 22
 
 ```sh
 # expose an RDP server on a specific public address that you reserved
-ngrok tcp --remote-addr=1.tcp.ngrok.io:27210 3389
+ngrok tcp --url=1.tcp.ngrok.io:27210 3389
```

Comment:
@KristopherPaulsen I remember seeing a comment about requiring scheme when doing TCP for some reason, is that something we should consider here?

Reply:
Yes! For now we still require explicitly adding the scheme for tcp (fix incoming soon™️).

eg) `--url=tcp://1.tcp.ngrok.io:27210`

Reply:
Yep, that was my miss.

Reply:
@nijikokun Fixed it (I think)